src/controllers/User/UserSignUpController.ts

```diff
@@ -0,0 +1,18 @@
+import * as jwt from 'jsonwebtoken';
+import * as bcrypt from 'bcrypt';
+
+import * as UserController from './UserController';
+import config from '../../config';
+import models from '../../models/db';
+
+/**
+ * ...
+ */
+export const signUpUser = async (username, password, email) => {
+  let encodedPw = bcrypt.hashSync(password, config.auth.HASH_SALT);
```

Comment:
This could be embedded inside the object notation, or, if not modified further down, should be defined with 'const'.

```
e.g. {
  ...
  password: bcrypt.hashSync(...), 
}
```

You @jcooky may have not taken charge of modifying this file, or did not get the chance to see this file. If that is the case, you may just leave it as is. This will be taken care of later on.